tests/boot/boot_to_desktop.pm

```diff
@@ -24,6 +24,7 @@ sub run {
     # We have tests that boot from HDD and wait for DVD boot menu's timeout, so
     # the timeout here must cover it. UEFI DVD adds some 60 seconds on top.
     my $timeout = get_var('UEFI') ? 140 : 80;
+    my $ready_time = get_var('USE_SUPPORT_SERVER_PXE_CUSTOMKERNEL') ? 900 : 500;
     # Increase timeout on ipmi bare metal backend, firmware initialization takes
```

Comment:
Maybe we want to wait a bit more (twice as much?), for those times when the network is loaded (which is kinda proved by how many boot_to_desktop failed across).

https://openqa.suse.de/tests/8576312#step/boot_to_desktop/10

Reply:
No, because longer timeout will end with black screen. Test you added used 500 timeout.